concat-kdf/src/lib.rs

```diff
@@ -0,0 +1,81 @@
+#![no_std]
+
+use core::fmt;
+use digest::{generic_array::typenum::Unsigned, Digest, FixedOutputReset, Update};
+
+#[cfg(feature = "std")]
+extern crate std;
```

Comment:
Why is this needed?

Reply:
https://doc.rust-lang.org/reference/names/preludes.html#the-no_std-attribute

```
error[E0433]: failed to resolve: use of undeclared crate or module `std`
  --> concat-kdf/src/lib.rs:74:13
   |
74 | ) -> Result<std::vec::Vec<u8>, Error>
   |             ^^^ use of undeclared crate or module `std`
```

`no_std` attribute prevents the `std` crate from being automatically added into scope.